### PR TITLE
[Controller] Add documentation

### PR DESCRIPTION
**Pull request by VasiliyPetrov1**
Solves Issue #105

### CODE REVIEW

**Comment by MLGMAG** [comment]
Make 1 commit from 2

File: src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java [hunk 1]
```diff
@@ -34,77 +44,187 @@ public void init(View view) {
...
 
+    /**
+     * This method removes {@link Event}
+     * <p>
+     * Receives instance of Event, calls {@link EventService#removeEvent(Event)} and puts received instance to it
+     *
+     * @param event
+     * @see EventService
+     */
     public void removeEvent(Event event) {
```
Comment (makrushinoff) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java:113]: This method doesn`t remove Event. It executes deletion on DB

File: src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java [hunk 2]
```diff
@@ -34,77 +44,187 @@ public void init(View view) {
...
 
+    /**
+     * This method sorts all existing events {@link Event} by date
+     * <p>
+     * Calls {@link CalendarService#sortEventsByDate(List)} and returns sorted list of events
+     *
+     * @param sortedList
+     * @return list of events sorted by date
+     * @see CalendarService
+     */
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java:200]: ... sorts all existing ...
All received Events  (all events in given list)

File: src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java [hunk 3]
```diff
@@ -34,77 +44,187 @@ public void init(View view) {
...
         final CalendarService currentCalendarService = getCalendarService();
         return currentCalendarService.sortEventsByPriority();
     }
 
+    /**
+     * This method gets instance of EventService{@link EventService}
+     *
+     * @return instance of EventService
+     * @see EventService
+     */
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/controller/Controller.java:186]: Redundant JavaDocs;
Don't write JavaDocs on setter or getters;
Put that method on the end of class

**Review by makrushinoff — CHANGES_REQUESTED** [review]

**Review by MLGMAG — CHANGES_REQUESTED** [review]

**Comment by makrushinoff** [comment]
Add javadoc to Controller class on line 15, with @author annotation and full class description

**Pull request closed without merging**